### PR TITLE
Add l.shenron.su

### PR DESCRIPTION
**Pull request by ShenronProxy**

### CODE REVIEW

File: long-live-pomf.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 | https://mixtape.moe/       | Up               | https://gitgud.io/Sapphire/mixtape.moe        |              100 | Pastebin, voice
 | https://safe.moe/          | Up               | https://github.com/WeebDev/loli-safe          |              200 | Pastebin
 | https://sub.god.jp/        | Up               | https://github.com/xatasan/registrars         |               32 | JS-free, pastebin
+| https://plexle.fi          | Up               | https://github.com/pomf/pomf                  |               20 | Zero Logs.
```
Comment (tsdko) [inline long-live-pomf.md]: Looks like a pastebin site. Pretty sure it doesn't run `https://github.com/pomf/pomf`.

Comment (ShenronProxy) [reply long-live-pomf.md]: Whoops sorry I meant to put https://l.shenron.su

File: long-live-pomf.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 | https://mixtape.moe/       | Up               | https://gitgud.io/Sapphire/mixtape.moe        |              100 | Pastebin, voice
 | https://safe.moe/          | Up               | https://github.com/WeebDev/loli-safe          |              200 | Pastebin
 | https://sub.god.jp/        | Up               | https://github.com/xatasan/registrars         |               32 | JS-free, pastebin
-| https://plexle.fi          | Up               | https://github.com/pomf/pomf                  |               20 | Zero Logs.               
+| https://l.shenron.su       | Up               | https://github.com/pomf/pomf                  |               20 | Zero Logs.
```
Comment (tsdko) [inline long-live-pomf.md]: The `source code` column is meant to link to the source code of the particular instance (with all customizations etc), not the codebase it was based on.

**Comment by tsudoko** [comment]
> Files automatically delete after 2 days.

This document is meant to list pomf.se alternatives. One important property of pomf.se was the ability to serve uploaded files indefinitely.

**Pull request closed without merging**